plugins/version-control-systems/git/src/main/kotlin/Git.kt

```diff
@@ -60,14 +63,50 @@ import org.ossreviewtoolkit.utils.ort.showStackTrace
 import org.semver4j.RangesList
 import org.semver4j.RangesListFactory
 
-// TODO: Make this configurable.
-const val GIT_HISTORY_DEPTH = 50
-
 // Replace prefixes of Git submodule repository URLs.
 private val REPOSITORY_URL_PREFIX_REPLACEMENTS = listOf(
     "git://" to "https://"
 )
 
+enum class OptionKey(val key: String, val defaultValue: String, val deprecated: Boolean = false) {
```

Comment:
All other plugins that take options just use constants for key names, and I believe we should do the same here, for consistency and to reduce complexity. I guess this will get reworked anyway once VCS plugins get migrated to @mnonnenmacher's new plugin API.

Reply:
Just saw it for the Gradle and GradleInspector plugins. Makes sense to do it in the same way.